README.md

```diff
@@ -9,15 +9,19 @@ Count and sum the scores of a bowling game for one player. For this challenge, y
 
 A bowling game consists of 10 frames in which the player tries to knock down the 10 pins. In every frame the player can roll one or two times. The actual number depends on strikes and spares. The score of a frame is the number of knocked down pins plus bonuses for strikes and spares. After every frame the 10 pins are reset.
 
-USER STORIES
+## DIAGRAM
+
+![Bowling Game Diagram](./images/bowling_game_diagram.png)
```

Comment:
This is one of the best diagrams i have ever seen so far! I imagine even a person who might never have played bowling before, could probably follow this diagram and grasp the logic / concept of the game! 10/10!!